src/resources/files.json

```diff
@@ -13,7 +13,7 @@
IBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Documentable<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>false<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Mitigations<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Potential_Impact<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Third_Party_Tools<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Mitigation_Control<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Responsibility<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Security_Override_Guidance<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Check_Content_Ref<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>M<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Weight<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>10.0<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>Class<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>{{classification}}<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>STIGRef<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>{{profileName}} {{startTime}}<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>TargetKey<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>{{target_key}}<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>STIG_UUID<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>{{uuidV4}}<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>LEGACY_ID<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>LEGACY_ID<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA><\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t{{#ccis}}\r\n\t\t\t\t<STIG_DATA>\r\n\t\t\t\t\t<VULN_ATTRIBUTE>CCI_REF<\/VULN_ATTRIBUTE>\r\n\t\t\t\t\t<ATTRIBUTE_DATA>{{.}}<\/ATTRIBUTE_DATA>\r\n\t\t\t\t<\/STIG_DATA>\r\n\t\t\t\t{{\/ccis}}\r\n\t\t\t\t<STATUS>{{status}}<\/STATUS>\r\n\t\t\t\t<FINDING_DETAILS>{{results}}<\/FINDING_DETAILS>\r\n\t\t\t\t<COMMENTS><\/COMMENTS>\r\n\t\t\t\t<SEVERITY_OVERRIDE><\/SEVERITY_OVERRIDE>\r\n\t\t\t\t<SEVERITY_JUSTIFICATION><\/SEVERITY_JUSTIFICATION>\r\n\t\t\t<\/VULN>\r\n\t\t\t{{\/controls}}\r\n\t\t<\/iSTIG>\r\n\t<\/STIGS>\r\n<\/CHECKLIST>"
```

Comment:
The format here is making me realize some of the things we need to look out for in the reverse mapper such as CCI_REF is typically only one instance and is the CCI_#. StigViewer doesn't break with multiple elements with the same name (a failing in the schema IMO) but just takes the last element with the name. Other things such as having a default CLASS because neither SAF CLI nor Heimdall parses `classification` as a variable from the HDF. I think the 'meat' of the information required is there, but a finer look will need to be used when reversing back in the future.